pkg/releasetesting/test_suite_test.go

```diff
@@ -328,6 +360,26 @@ func (p *podSucceededKubeClient) WaitAndGetCompletedPodPhase(ns string, r io.Rea
 	return core.PodSucceeded, nil
 }
 
+// For testing parallelism, this kube client
```

Comment:
For my own understanding, do you mind explaining why it is necessary for the first test to finish second if run in parallel?

Reply:
This isn't to ensure the first test finished second, it's so that the second test starts before the first test finishes, ensuring that they are indeed being run in parallel